package.json

```diff
@@ -0,0 +1,258 @@
+{
+  "name": "potential-robot",
+  "version": "1.0.0",
+  "type":"module",
```

Comment:
this line breaks `npm run dev` that is used in NFT-minter for running the project in a browser

Reply:
https://github.com/aleeusgr/potential-robot/issues/23